modules/genomic_browser/ajax/genomic_file_upload.php

```diff
@@ -139,6 +199,8 @@ function moveFileToFS(&$fileToUpload)
             )
         );
     }
+    // update i to success if no problems encountered
+    reportProgress(99, "File copied to $genomic_data_dir ");
```

Comment:
I'm also not sure if we want to be showing the user the full server side path here (or I may be misunderstanding what "reportProgress" does..)